portfolio/src/main/webapp/style.css

```diff
@@ -17,9 +53,45 @@
 }
 
 button {
+    display: flex;
+    border-radius: 8px;
+    padding: 20px;
+    max-width: 450px;
+    max-height: 150px;
+    background-color: rgb(248, 80, 80);
+    color: rgb(218, 217, 217);
+    text-align: center;
+    margin: 0px;
     cursor: pointer;
-    margin-bottom: 10px;
-    margin-top: auto;
+    transition: all 0.3s;
+    font-size: 20px;
+    font-family: "Comic Sans MS", cursive, sans-serif;
+    font-weight: bold;
+}
+
+button span {
+    transition: 0.3s;
+    cursor: pointer;
+    position: relative;
+    display: inline-block;
+}
+
+button span:after {
```

Comment:
The button on the main page is missing the inner `span` that makes this rule work.

However, if you want to have the same styling across multiple pages, perhaps those common rules could be moved into shared CSS file that is included on all pages, along with a second, page-specific CSS file.

Reply:
Ahh, my mistake. I forgot to add the inner span to the button in the index.html. I will add it to the index.html in the next commit.